gen/nvml/const_gen.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nvml
```

Comment:
Should this file to be present in the `pkg` folder too?

Reply:
Yes. It is there in my local setup (which is why my local tests passed), but since `pkg` is in `.gitignore` it didn't pick it up when I added the `pkg` folder to git -- I needed `git add pkg -f`.  Now I see why we should remove this from `.gitignore` :)

Updated.

Reply:
👍